bhclub.html

```diff
@@ -52,7 +53,26 @@ <h1>Events</h1>
             <h1>Teams</h1>
         </section>
         <section id="contact">
-            <h1>Contact</h1>
+            <div>
+                <h1>Contact</h1>
+                <p>Have any questions? Feel free to reach out to us at <a href="mailto:hello@brickhack.org">hello@brickhack.org</a></p>
+                <p>You can also sign up to receive newsletters <a href="./index.html">here</a>.</p>
```

Comment:
Not important yet, but do we have a plan for what to do after `index.html` stops being a newsletter form and turns into a full event site?

Reply:
Design is having me move the newsletter form directly to the page

This won't solve the issue with the newsletter being for the vent rather than the club, but I think I may try to clear up our wording a bit